etl-pipeline/api/app.py

```diff
@@ -28,32 +28,40 @@
 
 logger = create_log(__name__)
 
+BLUEPRINTS = [
+    search,
+    work,
+    works,
+    info,
+    edition,
+    editions,
+    utils,
+    link,
+    links,
+    opds,
+    collection,
+    collections,
+    citation,
+    fulfill,
+]
 
-class FlaskAPI:
-    def __init__(self, dbEngine, redisClient):
+
+class API:
```

Comment:
It's not good practice to use the underlying api framework in the naming. We could switch to a different framework, which now will require fewer changes.